tests/jvm/src/test/scala/scala/meta/tests/semanticdb/ExpectSuite.scala

```diff
@@ -50,12 +55,29 @@ trait ExpectHelpers {
     Files.write(path, value.getBytes(UTF_8))
 }
 
+object ConvertExpect extends ExpectHelpers {
+  def filename: String = "convert.expect"
+
+  def loadObtained: String = {
+    val classpath = BuildInfo.databaseClasspath
+    val target = Files.createTempDirectory("target_")
+    val metacp_settings = MetacpSettings.parse(List("-d", target.toString, classpath)).get
+    val (metacp_exitcode, _) = CliSuite.communicate(Metacp.process(metacp_settings))
+    assert(metacp_exitcode == 0)
+    val paths = Files.walk(target).iterator.asScala
+    val semanticdbs = paths.filter(_.toFile.isFile).map(_.toString).toArray
+    val (metap_exitcode, metap_stdout) = CliSuite.communicate(Metap.process(semanticdbs))
+    assert(metap_exitcode == 0)
+    metap_stdout
```

Comment:
Alternatively, we can add another `metacp-metap-diff.expect` that is the unified diff between the two for the for the symbols that metacp converts. In theory, that file should be empty but if there are known differences between the two then it will be highlighted here.